android/CouchbaseLite/build.gradle

```diff
@@ -67,6 +67,11 @@ def getGitHash = { ->
     return stdout.toString().trim()
 }
 
+// defining this property in the file ~/.gradle/gradle.properties
```

Comment:
How does this work on Jenkins?

Reply:
It should do exactly what it did before, except now it will produce an `x86_64` version as well.